src/middlewared/middlewared/plugins/jail.py

```diff
@@ -405,13 +405,13 @@ def common_validation(self, verrors, options, update=False, jail=None):
 
         return verrors
 
-    @accepts(Str("jail"))
-    def do_delete(self, jail):
+    @accepts(Str('jail'), Bool('force', default=False))
```

Comment:
I would rather we have a Dict() here with force in it.

Best for future backward compatibility.